### PR TITLE
compute: Update command to show expected output

### PR DESCRIPTION
**Pull request by MekalBoy**
Calling `strace -e clone ./sum_array_threads 2` would not show the clone3 syscall. Replacing `clone` with `clone,clone3` in both commands allows the syscalls to display as expected.

A small typo has also been fixed.

### CODE REVIEW

**Review by teodutu — CHANGES_REQUESTED** [review]
Thanks @MekalBoy. I suggested some inline fixes. Besides those, please reformat your commit description to 75 characters [1]. The super-linter failure is a false positive [2].

After this, please change the base branch of this PR to target `cs-pub-ro:main` instead of `open-education-hub:main`. The former repo is the one we're currently using and we'll push all updates from the latter into it at the end of the semester. Now to change the target, do the following:

```
# Add the `cs-pub-ro` repo as a remote
git remote add upb git@github.com:cs-pub-ro/operating-systems.git

# Feth the updates from the cs-pub-ro repo
git fetch upb

# Create a new branch that tracks cs-pub-ro:main
git check -b upb-main --track upb/main

# Move to the branch with the current fixes and rebase upb-main
git checkout computefix
git rebase -i upb-main
```

The last command will open your default editor with a list of commits. These are the extra commits from `cs-pub-ro:main` that haven't been integrated into `open-education-hub:main` yet. Change the word `pick` into `d` at the start of each line **apart from the last one**, which is the commit you're trying to make. Then save and exit. There should be no conflicts, but if there are any, fix them (they can only be line mismatches I think) and then run `git add <the files in this PR>` and then `git rebase --continue`. With or without conflicts, after the rebase is done, use `git push -f` to change your upstream branch, then open a new PR under the `cs-pub-ro` repo [3]. I'll close this one once the new PR is made.

[1] https://github.com/open-education-hub/operating-systems/actions/runs/7084827252/job/19279872877?pr=345#step:3:573
[2] https://github.com/open-education-hub/operating-systems/actions/runs/7084827252/job/19279872270?pr=345#step:3:694
[3] https://github.com/cs-pub-ro/operating-systems

**Comment by MekalBoy** [comment]
Pull request for cs-pub-ro/operating-systems has been made here:
https://github.com/cs-pub-ro/operating-systems/pull/36

File: content/compute/lab/content/arena.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ For this, we'll run both `sum_array_threads` and `sum_array_processes` under `st
 As we've already established, we're only interested in the `clone` syscall:
 
 ```console
-student@os:~/.../lab/support/sum-array/c$ strace -e clone ./sum_array_threads 2
+student@os:~/.../lab/support/sum-array/c$ strace -e clone,clone3 rray_threads 2
```
[the remark this answers is not in the document]
Comment (MekalBoy) [reply content/compute/lab/content/arena.md]: Addressed in latest commit: https://github.com/open-education-hub/operating-systems/pull/345/commits/3e3f609f5d08022d0d9986b4ecba54bb28edf480

File: content/compute/lab/content/arena.md [hunk 2]
```diff
@@ -8,17 +8,17 @@ For this, we'll run both `sum_array_threads` and `sum_array_processes` under `st
...
 clone(child_stack=0x7f60b56482b0, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, parent_tid=[1819693], tls=0x7f60b5649640, child_tidptr=0x7f60b5649910) = 1819693
 clone(child_stack=0x7f60b4e472b0, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, parent_tid=[1819694], tls=0x7f60b4e48640, child_tidptr=0x7f60b4e48910) = 1819694
 
-student@os:~/.../lab/support/sum-array/c$ strace -e clone ./sum_array_processes 2
+student@os:~/.../lab/support/sum-array/c$ strace -e clone,clone3array_processes 2
```
[the remark this answers is not in the document]
Comment (MekalBoy) [reply content/compute/lab/content/arena.md]: Addressed in latest commit: https://github.com/open-education-hub/operating-systems/pull/345/commits/3e3f609f5d08022d0d9986b4ecba54bb28edf480

**Pull request closed without merging**